Dockerfile

```diff
@@ -18,13 +20,13 @@ RUN yum install -y \
     # Dependencies for building xen \
     checkpolicy gcc python38 python38-devel iasl ncurses-devel libuuid-devel glib2-devel \
     pixman-devel selinux-policy-devel yajl-devel systemd-devel \
-    glibc-devel.i686 glibc-devel \
+    glibc-devel.i686 glibc-devel flex bison wget \
     \
     # Dependencies for building qemu \
     git libfdt-devel zlib-devel bzip2 \
     \
     # Crucible build dependencies \
-    rpm-build squashfs-tools openssl-devel rsync \
+    rpm-build squashfs-tools openssl-devel rsync python2 \
```

Comment:
I'm ok with this for now, but I'd like to move in the direction of killing any dependency we have on python2

Reply:
right now the main dependency that I've seen is download artifacts